db/event_handler.go

```diff
@@ -92,7 +92,7 @@ func (wh *Webhook) HandleEvent(event Event) {
 		// for DocumentChangeEvent, post document body
 		jsonOut, err := json.Marshal(event.Doc)
 		if err != nil {
-			base.Warn("Error marshalling doc for webhook post")
+			base.WarnR("Error marshalling doc for webhook post")
```

Comment:
Not strictly related, so feel free to ignore. But perhaps we could include the error in this log message, and the few below while we're here?

Reply:
Fixed